api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2MirrorSpec.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"sourceCluster", "targetCluster", "sourceConnector", "checkpointConnector", "heartbeatConnector", "topics", "groups"})
+@EqualsAndHashCode
+public class KafkaMirrorMaker2MirrorSpec implements Serializable, UnknownPropertyPreserving {
+    private static final long serialVersionUID = 1L;
+
+    private String sourceCluster;
+    private String targetCluster;
+    private KafkaConnectorSpec sourceConnector;
+    private KafkaConnectorSpec checkpointConnector;
+    private KafkaConnectorSpec heartbeatConnector;
+    private String topics;
+    private String groups;
+    private Map<String, Object> additionalProperties;
+
+    @Description("A regular expression matching the topics to be mirrored.")
+    @JsonProperty(required = true)
+    public String getTopics() {
```

Comment:
Wouldn't `topicsPattern` or `topicsRegex` be a better name for this property? To make clear its regular expression nature?

Reply:
Updated